src/Psr7ServerRequest.php

```diff
@@ -124,21 +124,18 @@ private static function convertUploadedFiles(array $uploadedFiles)
      */
     private static function convertFilesToUploaded(array $files)
     {
-        if (!isset($files['file'])) {
```

Comment:
Removed because I don't know why this is needed. maybe I am wrong here @weierophinney for what reason this is here?